tests/Integration/ApplicationTest.php

```diff
@@ -10,12 +10,12 @@
 
 class ApplicationTest extends IntegrationTestCase
 {
-    public function setUp()
+    protected function setUp(): void
     {
         $this->workspace()->reset();
     }
 
-    public function tearDown()
+    protected function tearDownt (): void
```

Comment:
Typo or automated process ?
```suggestion
    protected function tearDown (): void
```

Reply:
Automated typo :)